src/react-app-env.d.ts

```diff
@@ -87,12 +87,27 @@ interface PlayerDetails {
   bids: ItemBid[];
   auctions: Auction[];
 }
+enum FilterType
```

Comment:
remove this (it should be in another file after the previous task)

Reply:
leads to compiler errors.

Reply:
can't be removed the type below depends on it. 
And react-app.env.d.ts can't import other files